pkg/apis/kf/v1alpha1/space_types.go

```diff
@@ -53,6 +53,8 @@ type SpaceSpec struct {
 	// Execution contains settings for the execution environment.
 	// +optional
 	Execution SpaceSpecExecution `json:"execution,omitempty"`
+
+	ResourceLimits SpaceSpecResourceLimits `json:"resourceLimits,omitempty"`
```

Comment:
WDYT about moving the properties for limits under the Execution field?

Reply:
I think it makes more sense to have resource limits separate from Execution bc it seems more permanent than env variables/runtime configs. That said, I'm open to moving it in the future if there's a good case for it